GOVERNANCE.md

```diff
@@ -146,8 +146,31 @@ Contributions can be:
 
 ### Nominating a new Collaborator
 
-To nominate a new Collaborator, open an issue in the [nodejs/node][] repository.
-Provide a summary of the nominee's contributions. For example:
+To nominate a new Collaborator:
+
+1. **Optional but strongly recommended**: open a
+   [discussion in the nodejs/collaborators][] repository. Provide a summary of
+   the nominee's contributions (see below for an example).
+2. **Optional but strongly recommended**: After sufficient wait time (e.g. 72
+   hours), if the nomination proposal has received some support and no explicit
+   block, add a comment in the private discussion stating you're planning on
```

Comment:
This is also changing the nomination process in a way that requires I'm not comfortable with. If the nomination doesn't get enough attention because most collaborators are inactive, this will block node.js from introducing new collaborators.

If we are adding "if the nomination proposal has received some support" part, we should handle the part where there isn't enough support nor there isn't anybody blocking it.

The cost of explicitly defining these things, makes it easy to find a loophole or put yourself in a situation where you can not make a sane decision.

Reply:
It's an optional step, there's no way it could block the project